src/analysis/inner_product_space/positive.lean

```diff
@@ -124,3 +126,198 @@ end
 end complex
 
 end continuous_linear_map
+
+namespace linear_map
+
+open linear_map
+
+variables {V : Type*} [inner_product_space ℂ V]
+
+local notation `e` := is_symmetric.eigenvector_basis
+open_locale big_operators
+
+/-- `T` is (semi-definite) positive if `∀ x : V, ⟪x, T x⟫_ℂ ≥ 0` -/
+def is_positive (T : V →ₗ[ℂ] V) :
+  Prop := ∀ x : V, 0 ≤ ⟪x, T x⟫_ℂ.re ∧ (⟪x, T x⟫_ℂ.re : ℂ) = ⟪x, T x⟫_ℂ
+
+lemma is_self_adjoint_iff_real_inner [finite_dimensional ℂ V] (T : V →ₗ[ℂ] V) :
+  is_self_adjoint T ↔ ∀ x, (⟪x, T x⟫_ℂ.re : ℂ) = ⟪x, T x⟫_ℂ :=
+begin
+  simp_rw [← is_symmetric_iff_is_self_adjoint, is_symmetric_iff_inner_map_self_real,
+           inner_conj_sym, ← is_R_or_C.re_eq_complex_re, ← is_R_or_C.eq_conj_iff_re,
+           inner_conj_sym],
+  exact ⟨λ h x, (h x).symm, λ h x, (h x).symm⟩,
+end
```

Comment:
Could you move this next to `is_symmetric_iff_inner_map_self_real` and restate it in terms of `is_symmetric` (which allows you to remove the `finite_dimensional` assumption)? It may mean doing a rewrite between `is_self_adjoint` and `is_symmetric` more often, but morally when working with raw linear maps `is_symmetric` is the "right" notion, because it always exists.

Reply:
did this, but most of the results needed to be in a finite-dimensional setting